### PR TITLE
Standardize TargetFramwork for VisualStudio IDE

### PR DESCRIPTION
**Pull request by stan-sz**
`net5.0` and `net6.0` are the standard target framework monikers in VS. Additionally, use singular `<TargetFramework>` for a single TFM values.
This allows opening the sln file in VS IDE. Additionally made CRLF->LF changes and test framework upgrade.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,15 +141,15 @@ git clone https://github.com/kubernetes-client/gen
...
 ```
 
-# Version Compatibility 
+# Version Compatibility
 
 | SDK Version | Kubernetes Version | .NET Targeting                        |
 |-------------|--------------------|---------------------------------------|
-| 7.0         | 1.23               | netstandard2.1;net5;net6              |
+| 7.0         | 1.23               | netstandard2.1;net5.0;net6.0          |
 | 6.0         | 1.22               | netstandard2.1;net5                   |
```
Comment (tg123) [inline README.md:153]: seems still net5 here

BTW, any doc link for `net5.0` is recommend way? just curious.

File: src/KubernetesClient.Basic/KubernetesClient.Basic.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (tg123) [inline src/KubernetesClient.Basic/KubernetesClient.Basic.csproj:4]: would this cause VS issue?
seems my VS works well

**Review by tg123 — COMMENTED** [review]
/LGTM

some nits

File: README.md (hunk 1 above)
Comment (stan-sz) [reply README.md:153]: https://docs.microsoft.com/en-us/dotnet/standard/frameworks

File: src/KubernetesClient.Basic/KubernetesClient.Basic.csproj (hunk 2 above)
Comment (stan-sz) [reply src/KubernetesClient.Basic/KubernetesClient.Basic.csproj:4]: Yes. I've isolated this change locally and restarting VS solved the bogus errors problem.

Comment (stan-sz) [reply src/KubernetesClient.Basic/KubernetesClient.Basic.csproj:4]: Possibly related to https://github.com/dotnet/sdk/issues/8676?

File: README.md (hunk 1 above)
Comment (tg123) [inline README.md:153]:
```suggestion
| 6.0         | 1.22               | netstandard2.1;net5.0                 |
```

Comment (stan-sz) [reply README.md:153]: This PR will pertain only to 7.x packages, unless we port it to previous version, this I deliberately not modified this particular value.

Comment (stan-sz) [reply README.md:153]: But OK, I'll incorporate your suggestion.

**Pull request merged**